main.cc

```diff
@@ -622,27 +652,71 @@ do_getopt(int argc, char *argv[]) noexcept
 		case 'T':
 			g_opt.timeout = atoi(optarg);
 			break;
+		case 'V':
+			if (!strcmp(optarg, "1.2")) {
+				g_opt.tls_vers = TLS1_2_VERSION;
+			} else if (!strcmp(optarg, "1.3")) {
+				g_opt.tls_vers = TLS1_3_VERSION;
+			} else if (!strcmp(optarg, "any")) {
+				g_opt.tls_vers = TLS_ANY_VERSION;
```

Comment:
Not  big deal in this simple tool, but it'd be better to use `strncmp()` and take care about the `\0` string trailer.

Reply:
Fixed